examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<p>Add labels etc.</p><p> </p><p>Would be nice to add a comparison vs staggered 1st order acoustic (with a difference plot).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>This was useful, since it turns out this highlights the higher stability limit of ADER vs staggered</p>